bin/bl-beepmein-pipemenu

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# pipeBeepmein
+# beepmein pipe menu for jgmenu
+
+# usage: pipeBeepmein | jgmenu --simple --icon-size=0
+
+# required
+command -v at >/dev/null 2>&1 || { >&2 echo "I need at: sudo apt install --no-install-recommends at" ; exit 1; }
```

Comment:
Maybe loop. No install recommendation for gmrun?

Reply:
added gmrun.